### ISSUE TITLE
Transmitter Days x/60.1?

### ISSUE THREAD

**Issue by IntoxRRR** [NightscoutFoundation/xDrip#1485]
Recently I noticed a different display of values for "Transmitter Days". Before, it always showed just the days running, now it says "33/60.1". What does the "60.1" mean? I expect the transmitter to live for at least 90 days.

Version: 2020.07.13

![transmitter_days](https://user-images.githubusercontent.com/60309229/97065990-b8473080-15b1-11eb-9aad-249f4f2ccfec.jpg)

**Comment by tolot27** [comment]
Did the value after the slash decreases while the value before the slash increases? My guess is that it shows the remaining days.

**Comment by emp-00** [comment]
I'm on exactly the same xDrip+ version: 2020.07.13 and also running an 8J tx with exactly the same firmware.
==> only showing a single value here, obviously the number of days the tx is already in operation.

**Comment by IntoxRRR** [comment]
The value after the slash stays the same. Or it stopped decreasing, if showing the remaining days is its purpose.
The first value is counting correctly: now "34/60.1". I’m pretty sure there was only the single value at start.

To add the latest circumstances:
I did a restart of the previous sensor after ten days, with a rechargeable 81xxx transmitter for testing purpose. After five days I stopped it and started a new sensor, again with the 8J transmitter. Maybe switching the transmitters caused this? Unfortunately I cant’t remember when the display changed; _while_ using the 81 or _after_ going back to the 8J.
